docs/advanced-features/s3-prefix-permissions.md

```diff
@@ -0,0 +1,34 @@
+# Restricting S3 access to specific prefix in bucket
```

Comment:
"Restricting S3 access to specific prefix in bucket" change to "Restricting access to specific prefixes in a S3 bucket"